scripts/apb-docker-run.sh

```diff
@@ -16,6 +16,7 @@ if ! [[ -z "${DOCKER_CERT_PATH}" ]] && [[ ${DOCKER_CERT_PATH} = *"minishift"* ]]
 fi
 
 KUBECONFIG_ENV="${KUBECONFIG:+-v ${KUBECONFIG}:${KUBECONFIG} -e KUBECONFIG=${KUBECONFIG}}"
+[[ -z $KUBECONFIG ]] && KUBECONFIG_ENV="${KUBECONFIG:--v $HOME/.kube:/.kube -e KUBECONFIG=/.kube/config}"
```

Comment:
Maybe even better, remove this, and just set KUBECONFIG in line 17, if its not set yet:
```
KUBECONFIG=${KUBECONFIG:-/.kube/config}
```